hack/jenkins/minikube_cross_build_and_upload.sh

```diff
@@ -41,6 +41,7 @@ declare -rx DEB_VER="$(make deb_version)"
 docker kill $(docker ps -q) || true
 docker rm $(docker ps -aq) || true
 docker system prune -a --volumes -f
+docker login -u ${DOCKERHUB_USER} -p ${DOCKERHUB_PASS}
```

Comment:
Can we make this more clear that this does not use a read/write password?


```
docker login -u ${DOCKERHUB_USER} -p ${DOCKERHUB_READONLY_TOKEN}
```

This will help the Jenkins admin to do the right thing.